requirements.txt

```diff
@@ -6,3 +6,4 @@ docplex==2.15.194; sys_platform == 'darwin'
 setuptools>=40.1.0
 networkx>=2.2
 dataclasses; python_version < '3.7'
+gurobipy
```

Comment:
Gurobipy should be an extra library. So, this line needs to move `setup.py` similar to `cplex`.
https://github.com/Qiskit/qiskit-optimization/blob/f3a4df4e3fe75331b9475755462e5fae5e3254b8/setup.py#L64

Reply:
Done